config/crd/external/gateways.networking.istio.io.yaml

```diff
@@ -0,0 +1,258 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
why is this needed?
Doesn't istio installation provides it already?

Reply:
For the test, this is needed. The new reconciler updated KServe Local Gateway in the istio-system.

Reply:
so, it could be moved elsewhere?

Reply:
okay, is this packaged with the bundle image?

Reply:
nope